CIP-0165/namespaces/README.md

```diff
@@ -0,0 +1,18 @@
+# Namespaces
+
+This is directory of the supported namespaces.
+
+Each namespace defines a non-intersecting slices of the data.
+
+| Shortname | Content                         |
+| --------- | ------------------------------- |
+| utxo      | UTXOs                           |
+| stake     | Stake delegation                |
+| rewards   | Reward accounts                 |
+| params    | Protocol parameters             |
+| pots      | Accounting pots (reserves etc.) |
+| spo       | SPO state                       |
+| drep      | DRep state                      |
+| gov       | Governance action state         |
```

Comment:
Governance state has proposals which are stored in a forest with trees like structure. From, what I understand there was only a mention of flat maps and arrays like data structures being stored